25.10/MyCoolProj2/Controllers/BecauseICan.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.IO;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using MyCoolProj2.Interfaces;
+
+namespace MyCoolProj2.Controllers
+{
+    [ApiController]
+    [Route("App/[controller]")]
+    public class BecauseICan : Controller
```

Comment:
Controllers must have 'Controller' postfix 
this class and file should be called 'BecauseICanController'